src/server/index.ts

```diff
@@ -13,48 +12,43 @@ const {
   API_PORT,
   EXECUTION_SERVICE_URI,
   WALLET_SERVICE_URI,
-  CONTRACT_SERVER_LOWER_PORT_BOUND,
-  CONTRACT_SERVER_UPPER_PORT_BOUND,
   OPERATION_MODE,
   REDIS_HOST,
-  REDIS_PORT
+  REDIS_PORT,
+  CONTRACT_DIRECTORY
 } = process.env
 
 if (
   !API_PORT ||
   !EXECUTION_SERVICE_URI ||
   !WALLET_SERVICE_URI ||
-  !CONTRACT_SERVER_LOWER_PORT_BOUND ||
-  !CONTRACT_SERVER_UPPER_PORT_BOUND ||
-  !OPERATION_MODE
+  !OPERATION_MODE ||
+  !CONTRACT_DIRECTORY
 ) {
   throw new Error('Required ENVs not set')
 }
 
-const networkInterface = axios.create()
+// Allow 200mb contract images
+const networkInterface = axios.create({ maxContentLength: 200000000 })
```

Comment:
Happy with this. We can easily lift to config if wanting to lock it down further